google/metadata.go

```diff
@@ -161,9 +137,9 @@ func flattenMetadata(metadata *computeBeta.Metadata) map[string]string {
 
 // This function differs from flattenMetadata only in that it takes
 // compute.metadata rather than computeBeta.metadata as an argument. It should
-// be removed in favour of flattenMetadata if/when this resource gets beta
-// support.
-func flattenCommonInstanceMetadata(metadata *compute.Metadata) map[string]string {
+// be removed in favour of flattenMetadata if/when all resources using it get
+// beta support.
```

Comment:
I think this comment is meant for the `flattenMetadataBeta` func above.

Reply:
Ah nope it was meant for this one but just referred to the other one wrong. Fixed.